lib/database/memoryDB.go

```diff
@@ -61,7 +61,7 @@ func (db *MemDatabase) Get(k []byte) ([]byte, error) {
 	if v, ok := db.db[string(k)]; ok {
 		return v, nil
 	}
-	return nil, errors.New("not found")
+	return nil, errors.New("Key not found")
```

Comment:
standard error format starts with lowercase :eyes:

Reply:
this is done to match the badger DB error that's returned

Reply:
maybe a comment so someone is not tempted to update it in the future